README.md

```diff
@@ -22,21 +20,85 @@ Or install it yourself as:
 
 ## Usage
 
-TODO: Write usage instructions here
+Running `rubycritic_extension -b base_branch, feature_branch` will analyse all the Ruby files in the base_branch and the feature_branch and compare the score between the two branches, before running this commit all your changes in both the branches:
+
+```bash
+$ rubycritic_extension -b base_branch,feature_branch
+```
+
+Alternatively if you're using GitLab you can pass `pull_request_id` as the third argument which will post a comment to the respective pull_request:
+
+```bash
+$ rubycritic_extension -b base_branch,feature_branch,pull_request_id
```

Comment:
parameters should be space separated not comma separated (like unix commands)
eg.
```
rubycritic-ext -b base_branch feature_branch -p pull_request_id
```